.github/workflows/enterprise.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         with:
           ruby-version: '2.6'
       - name: Set up JDK 11
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
```

Comment:
This won't work, you must specify `distribution` as well, see https://github.com/actions/setup-java#usage